src/Microsoft.Health.Dicom.Core/Features/Validation/ValidationErrorCode.cs

```diff
@@ -0,0 +1,69 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Features.Validation
+{
+    /// <summary>
+    /// Validation Error Code.
+    /// Error Code is a short number (from 0 to 32767), and organized in this way:
+    /// 0 => No error.
+    /// [1,100) => General error for all VR.
+    /// [100,200) => Error for PN.
+    /// [200,300) => Error for DA.
+    /// Each VR could have up to 100 error code.
+    /// </summary>
+    public enum ValidationErrorCode
+    {
+        /// <summary>
+        /// No error.
+        /// </summary>
+        None = 0,
+
+        /// <summary>
+        /// Dicom element has multiple values -- we only support indexing single value dicom element.
+        /// </summary>
+        ElementHasMultipleValues = 1,
```

Comment:
If we really need to use errorCodes, we need strongly typed exceptions with the error code in them

Reply:
The error code will be stored into SQL database.  we won't throw exceptions based on them.